### PR TITLE
Add packages

### PR DESCRIPTION
**Pull request by cverna**
This PR creates a new components app, this is used to
store components built in Fedora (rpms, modules, containers, flatpaks, ...)
The first component added is RPMPackage.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,9 +19,9 @@ prune:
 	sudo podman volume rm -f pgdata
 
 up:
-	sudo podman pod create --name fpdc-dev -p 8000:8000
-	sudo podman run --name fpdc-web --pod fpdc-dev -dt -e OIDC_RP_CLIENT_ID=$(OIDC_RP_CLIENT_ID) -e OIDC_RP_CLIENT_SECRET=$(OIDC_RP_CLIENT_SECRET) -e DJANGO_SETTINGS_MODULE=fpdc.settings.base -v $(CURDIR):/code fpdc-web
-	sudo podman run --name database --pod fpdc-dev -dt -e PGDATA=/var/lib/postgresql/data/pgdata -v pgdata:/var/lib/postgresql/data/pgdata  postgres
+	sudo podman pod create --name fpdc-dev
+	sudo podman run --name fpdc-web --net=host --pod fpdc-dev -dt -e OIDC_RP_CLIENT_ID=$(OIDC_RP_CLIENT_ID) -e OIDC_RP_CLIENT_SECRET=$(OIDC_RP_CLIENT_SECRET) -e DJANGO_SETTINGS_MODULE=fpdc.settings.base -v $(CURDIR):/code fpdc-web
+	sudo podman run --name database --net=host --pod fpdc-dev -dt -e PGDATA=/var/lib/postgresql/data/pgdata -v pgdata:/var/lib/postgresql/data/pgdata  postgres
```
Comment (pypingou) [inline Makefile:24]: I thought one of the advantage of podman was to not require root access?

**Review by pypingou — APPROVED** [review]
Over all, looks fine to me

File: Makefile (hunk 1 above)
Comment (cverna) [reply Makefile:24]: Yes but the `pod` functionality is not yet available in rootless mode this is a fairly new feature which was just released so rootless support for pods might come in future releases

**Comment by cverna** [comment]
Thanks

**Pull request merged**